setup.py

```diff
@@ -24,11 +27,26 @@ def is_requirement(line):
     """
     return line and not line.startswith(('-r', '#', '-e', 'git+', '-c'))
 
+
 setup(
     name="chem",
-    version='1.0.0',
+    description='openedx chem',
```

Comment:
Maybe use the description from the README instead? "A helper library for chemistry calculations, used by the edx-platform"